benchmarks/nso-en/jw300-baseline/README.md

```diff
@@ -0,0 +1,46 @@
+# Sepedi to English
+
+Author: Thabo Makola
+
+## Data
+
+	- The JW300 English-Sepedi.
+
+## Model
+
+	- Default Masakhane Transformer translation model.
+	- Link to google drive folder with model(https://drive.google.com/drive/folders/1Mr0EHbXpBXKgPTDKdKfe4sLxVy94-8JU?usp=sharing)
+
+## Analysis
+
+Example 1
+```sh
+	Source: Gantši ba feleletša ba lemoga gore re dira mošomo o mobotse .
+ 	Reference: Often , the result is that our activity is seen in a positive way .
+ 	Hypothesis: They often realize that we do good work .
+```
+
+Example 2
+```sh
+	Source: Ge ma - Mexico a setlogo a belegetšwego moo a be a sa amogele bodumedi bjo bofsa , a be a tšewa e le barapedi ba diswantšho gomme a tlaišwa o šoro .
+ 	Reference: If native-born Mexicans did not embrace the new religion , they were regarded as idolaters and were severely persecuted .
+ 	Hypothesis: When Mexico was born there was no longer acceptable religion , he was taken as idolatry and suffered severe persecution .
+```
+
+Example 3
+```sh
+	Source: Go sa šetšwe gore o na le bana ba bakae , o se ke wa langwa go ba thušeng tseleng e išago bophelong bjo bo sa felego .
+ 	Reference: No matter how many children you have , never give up in helping them along the path to everlasting life .
+ 	Hypothesis: Whether you have a few children , do not be constructed to help them in a way to everlasting life .
+```
+
+Example 4
+```sh
+	Source: Moprofesara Rudolf Schenkel , mophologi wa tlhaselo ya tšhukudu yo go boletšwego ka yena pejana o lla ka therešo ya gore motho o itirile lenaba le nnoši leo tšhukudu e nago le lona .
+ 	Reference: Professor Rudolf Schenkel , the survivor of the rhino charge described earlier , laments the fact that man has made himself the only enemy the rhino has .
+ 	Hypothesis: Professor Rudolf Schenkel , a primarily relief mentioned earlier charges the fact that a person has a solid enemy .
+```
+
+# Results
+	- BLEU dev :  34.65
```

Comment:
This looks slightly better than #195, as you trained longer. Thanks for being so patient!